tools/extract_release_notes.rb

```diff
@@ -53,23 +55,24 @@ def replace_pr_link_with_markdown_link(string)
 release_lines = []
 lines[2...].each do |line|
   break if line.strip == ''
+
   release_lines.push line
 end
 
 case mode
 when :strip_pr_links
-  formatted_lines = release_lines.
-    map { |l| l.gsub(/ \[.*\]$/, '') }
+  formatted_lines = release_lines
+                    .map { |l| l.gsub(/ \[.*\]$/, '') }
 when :keep_pr_links
   formatted_lines = release_lines.
-    # The PR "links" are not canonical markdown links. That's not a problem on
-    # GitHub where they be automatically parsed into links to the corresponding
-    # PR, but outside GitHub, such as in our internal posts that might be
-    # confusing.
-    #
-    # It's probably best to update the convention in writing the release notes
-    # but in the meantime let's compensate with more automation.
-    map { |l| replace_pr_link_with_markdown_link(l) }
+                    # The PR "links" are not canonical markdown links. That's not a problem on
+                    # GitHub where they be automatically parsed into links to the corresponding
+                    # PR, but outside GitHub, such as in our internal posts that might be
+                    # confusing.
+                    #
+                    # It's probably best to update the convention in writing the release notes
+                    # but in the meantime let's compensate with more automation.
+                    map { |l| replace_pr_link_with_markdown_link(l) }
```

Comment:
Same note as the other PRs, moving the comment lines one above the whole expression so that we can then use:
```
# … the comment lines
formatted_lines = release_lines
  .map { |l| … }
```
And be consistent with code formatting used in previous `when` + reduce indentation.

Reply:
Fixed in https://github.com/Automattic/simplenote-android/commit/bfca4be9f80785456ef9dae4fae71d25728cb0e0